server/src/main/java/io/spine/server/command/CommandHandlingEntity.java

```diff
@@ -172,4 +176,70 @@ protected ValueMismatch expectedNotEmpty(String expected) {
     protected ValueMismatch unexpectedValue(String expected, String actual, String newValue) {
         return StringMismatch.unexpectedValue(expected, actual, newValue, versionNumber());
     }
+
+    /**
+     * A object which performs filtering of the events emitted by a
```

Comment:
That isn't an object yet. 
```A filter for events emitted by ... ```

Also there should be an explanation of why we have it.

Another idea I have (looking at its current usage) is that the ideal way of incorporating it into the flow would be the following. Right now it's possible to "forget" to invoke the filter. What if we force SPI users to do that (those who may want to extend the framework).

Let's discuss in person.